### PR TITLE
tranquilizer recipe ready

### PR DESCRIPTION
**Pull request by AlbertDeFusco**
@conda-forge/help-python This recipe is ready for review.

### CODE REVIEW

File: recipes/tranquilizer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - tranquilizer
+  commands:
+    - tranquilizer --help
+
+about:
+  home: http://github.com/AlbertDeFusco/tranquilizer
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```
Comment (chrisburr) [inline recipes/tranquilizer/meta.yaml]:
```suggestion
  license_file: LICENSE
```

File: recipes/tranquilizer/LICENSE [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Copyright (c) 2018, Anaconda, Inc.
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
```
Comment (chrisburr) [inline recipes/tranquilizer/LICENSE]: This file shouldn't be needed as it's already included in the source tarball.

File: recipes/tranquilizer/meta.yaml (hunk 1 above)
Comment (AlbertDeFusco) [reply recipes/tranquilizer/meta.yaml]: sure

File: recipes/tranquilizer/LICENSE (hunk 2 above)
Comment (AlbertDeFusco) [reply recipes/tranquilizer/LICENSE]: done

**Comment by AlbertDeFusco** [comment]
Is there something wrong with my recipe that is causing win64 builds to fail?

**Comment by chrisburr** [comment]
No that's an issue with `staged-recipes` (#7715). It's still tested on Windows using Appveyor.

**Comment by chrisburr** [comment]
I think this is ready to merge but do you mind if it is delayed slightly so I can use it to try and debug with Azure CI?

**Comment by AlbertDeFusco** [comment]
That's ok

**Pull request merged**